examples/rx_async.py

```diff
@@ -10,7 +10,4 @@ async def main():
 
```

Comment:
This will require something to keep the main-function from exiting, e.g.
```py
# keep coroutine alive
try:
    await asyncio.get_running_loop().create_future()
finally:
    ruuvi_rx.stop()
```

Reply:
I added this but I am not sure if it is correct. Please check.